query/src/main/scala/hmda/query/projections/filing/FilingCassandraProjection.scala

```diff
@@ -0,0 +1,31 @@
+package hmda.query.projections.filing
+
+import akka.NotUsed
+import akka.stream.alpakka.cassandra.scaladsl.CassandraSink
+import akka.stream.scaladsl.Source
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.persistence.messages.events.processing.CommonHmdaValidatorEvents.LarValidated
+import hmda.query.repository.filing.FilingCassandraRepository
+import hmda.persistence.processing.HmdaQuery._
+import hmda.query.model.filing.LoanApplicationRegisterQuery
+import hmda.query.repository.filing.LarConverter._
+
+class FilingCassandraProjection extends FilingCassandraRepository {
+
+  def startUp(): Unit = {
+    createKeyspace()
+    createTable()
+
+    val source: Source[LoanApplicationRegister, NotUsed] = liveEvents("HmdaFiling-2017").map {
```

Comment:
I'm seeing the 2017 year a lot throughout this PR,  are we OK with having this value hard-coded?